### DISCUSSION TITLE
Personal styles delete the styles of Tailwindcss

### DISCUSSION THREAD

**Discussion by maria1186** [tailwindlabs/tailwindcss#8861] [Help]
**What version of Tailwind CSS are you using?**

For example: v3.1.6

**What build tool (or framework if it abstracts the build tool) are you using?**

Pug cli

**What version of Node.js are you using?**

For example: v14.8.0

**What browser are you using?**

 Chrome

**What operating system are you using?**

Windows

**Reproduction URL**
https://github.com/maria1186/tailwind-issue

**Describe your issue**

When I try to add my own style, in the file where I imported the Tailwindcss styles, the compiled file deleted the last written code in the Tailwindcs itself. My style is there but not on Tailwindcs.  .hidden {
  display: none;
} then there is a deleted code which is as follows @media (min-width:768px). My goal was to add a class that would center align the text. Why the class given in Тailwindcss doesn't work .text-center. Some overwriting is occurring.

**Comment by adamwathan** [comment]
Hey thanks for including the code! I'm having a bit of trouble understanding exactly what steps to take to see the problem, can you give me step by step instructions to follow?

For example I'm not sure what you mean by "the compiled file deleted the last written code in the Tailwind CSS itself" — which code is being deleted?

**Comment by maria1186** [comment]
In folder src/style.css - in it I write the following code:  .text-align{text-align:center}.  
Before starting any editing of the files I run the following command: npx tailwindcss ./src/style.css -o ./public/style.css --watch. 
When I edit a file and save it, it automatically updates. The updated file is located in the public/style.css. 
I write in src/style.css .text-align:{text-align:center}, after save **part of code in public/style.css is deleted** - it is code of the framework itself. I hope I explained enough. Thanks for your quick response.

**Comment by maria1186** [comment]
I thought a lot about how to solve this overwriting, I made a number of attempts and finally it worked. 
I saved my class at the top of the file src/style.css 
 ` .text-center{
    text-align: center;
}

@tailwind base;
@tailwind components;
@tailwind utilities;`

So nothing is overwritten.
Here the documentation can be supplemented with my case [Tailwindcss](https://tailwindcss.com/docs/adding-custom-styles#customizing-your-theme)
